x-pack/plugins/triggers_actions_ui/public/application/sections/rules_list/components/rules_list.test.tsx

```diff
@@ -471,6 +498,7 @@ describe('rules_list component with items', () => {
     expect(wrapper.find('EuiHealth[data-test-subj="ruleStatus-pending"]').length).toEqual(1);
     expect(wrapper.find('EuiHealth[data-test-subj="ruleStatus-unknown"]').length).toEqual(0);
     expect(wrapper.find('EuiHealth[data-test-subj="ruleStatus-error"]').length).toEqual(2);
+    expect(wrapper.find('EuiHealth[data-test-subj="ruleStatus-warning"]').length).toEqual(1);
```

Comment:
We also need to add a `<EuiHealth>` badge to show the number of rules with warnings. This shows up right above the rule list table.

Reply:
Done :)